### PR TITLE
Allow withdrawing files from the file list view

### LINKED ISSUE #247: Withdraw files from the list view
https://bennettoxford.slack.com/archives/C065F8P58AW/p1712755051357289?thread_ts=1712753890.166379&cid=C065F8P58AW

> To drop a file from the request you have to do it from the individual #file view. Whilst i'm preparing a request, I'd like to be able to drop them from the list view.

### PR DESCRIPTION
**Pull request by evansd**
Closes #247 

UI preview:
image

### CODE REVIEW

File: airlock/views/request.py [hunk 1]
```diff
@@ -475,6 +483,39 @@ def file_withdraw(request, request_id, path: str):
...
+            try:
+                bll.withdraw_file_from_request(release_request, path, request.user)
+                successes.append(f"The file {path} has been withdrawn from the request")
+            except exceptions.RequestPermissionDenied as exc:
+                errors.append(str(exc))
+
+        if errors:
+            display_multiple_messages(request, errors, "error")
+        if successes:
+            display_multiple_messages(request, successes, "success")
```
Comment (madwort) [inline airlock/views/request.py:513]: for interest, I was just working near [the equivalent place in workspace.py](https://github.com/opensafely-core/airlock/blob/main/airlock/views/workspace.py#L330-L332), which has a slightly different behaviour - I think I prefer this though.

File: airlock/templates/file_browser/_includes/directory_table/request.html [hunk 2]
```diff
@@ -24,6 +29,13 @@
         <td>{{ path.request_filetype.name|title }}</td>
         <td data-order="{{ path.size }}">{{ path.size_mb }}</td>
         <td>{{ path.modified_at|date:"Y-m-d H:i" }}</td>
+        {% if multiselect_withdraw %}
+          <td>
+            {% if True %}
```
Comment (madwort) [inline airlock/templates/file_browser/_includes/directory_table/request.html:34]: why `if True`?

**Review by madwort — APPROVED** [review]

File: airlock/views/request.py (hunk 1 above)
Comment (evansd) [reply airlock/views/request.py:513]: Ah yeah, I hadn't seen that. I think it's better to keep them separate though.

File: airlock/templates/file_browser/_includes/directory_table/request.html (hunk 2 above)
Comment (evansd) [reply airlock/templates/file_browser/_includes/directory_table/request.html:34]: :facepalm: Leftover debugging! Thanks for catching

File: airlock/views/request.py (hunk 1 above)
Comment (madwort) [reply airlock/views/request.py:513]: once this has merged I think I'll change workspace to match

**Pull request merged**